.github/workflows/ci.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           name: actions ${{ matrix.node }}
-          fail_ci_if_error: true
+          fail_ci_if_error: false
   windows:
```

Comment:
Is this intentional?

Reply:
this is correct 👍 I switched the generator upstream and merged it.